### DISCUSSION TITLE
oleobj finds links in a document I cannot find

### DISCUSSION THREAD

**Discussion by Thanathan-k** [decalage2/oletools#801] [Q&A]
Hi all,

I have a document scanned with oleid and oleobj and they identified 3 links in the document:

`Found relationship 'hyperlink' with external link https://...`

However I was not able to find any of the links in both LibreOffice, as well as Excel using the search functionality.

Has anyone have any idea where these links might be stored and how to find/delete them from the document?

**Comment by decalage2** [comment]
Hi, would it be possible to share the sample, so that I have a look?
oleobj finds external relationships by parsing all XML files contained in the document, and looking for XML tags "Relationship" with TargetMode attribute = 'External'.
